src/app/services/locale.service.ts

```diff
@@ -1,12 +1,12 @@
 import { Injectable } from '@angular/core';
 import { CoreEvent } from 'app/interfaces/events';
 import { Option } from 'app/interfaces/option.interface';
```

Comment:
1. Selecting time format options with 'am' in settings does not format time correctly (for example in Task Manager).
2. Selecting 'am' options, saving and then reopening localization form will not populate Time Format with current value.